prodtests/full_system_test.sh

```diff
@@ -103,19 +103,19 @@ if [ "0$GENERATE_ITSMFT_DICTIONARIES" == "01" ]; then
 fi
 
 mkdir -p raw
-taskwrapper itsraw.log o2-its-digi2raw --file-for link  -o raw/ITS
-taskwrapper mftraw.log o2-mft-digi2raw --file-for link  -o raw/MFT
-taskwrapper ft0raw.log o2-ft0-digi2raw --file-per-link  -o raw/FT0
-taskwrapper fv0raw.log o2-fv0-digi2raw --file-per-link  -o raw/FV0
-taskwrapper fddraw.log o2-fdd-digit2raw --file-per-link  -o raw/FDD
-taskwrapper tpcraw.log o2-tpc-digits-to-rawzs  --file-for link  -i tpcdigits.root -o raw/TPC
+taskwrapper itsraw.log o2-its-digi2raw --file-for cru -o raw/ITS
+taskwrapper mftraw.log o2-mft-digi2raw --file-for link -o raw/MFT
+taskwrapper ft0raw.log o2-ft0-digi2raw --file-per-link -o raw/FT0
+taskwrapper fv0raw.log o2-fv0-digi2raw --file-per-link -o raw/FV0
+taskwrapper fddraw.log o2-fdd-digit2raw --file-per-link -o raw/FDD
+taskwrapper tpcraw.log o2-tpc-digits-to-rawzs --file-for cru -i tpcdigits.root -o raw/TPC
```

Comment:
with the changes above and `-o raw/TPC` you will now have TPC twice in the name. Perhaps simply change it to `-o raw/`